CONTRIBUTING.adoc

```diff
@@ -246,6 +246,27 @@ Be aware that some of this content might already be present in the Jenkins User
 
 The documentation pages can use the same metadata (`title`, `description`, `opengraph:image`) as blog posts.
 
+* When creating or adding documentation, make sure that every numbered line contains a complete sentence, including an ending punctuation mark (`.`, `?`, `!`).
+Formatting your documentation this way makes it easier for reviewers to read over and provide feedback. 
++
+----
+The full sentence structure in a line makes it easier to read.
+Otherwise, sentences are broken up over multiple lines or multiple sentences are combined into one line.
+In cases of code examples or output, this is not necessary and should be formatted normally.
+----
+
+* When creating documentation, avoid future tense as much as possible.
```

Comment:
Why is this a list?

Reply:
I've updated it to not be a list, that was more for me to separate topics, but it is not needed in the actual documentation. Thanks @daniel-beck!